_data/projects/100LinesOfCode.yml

```diff
@@ -0,0 +1,18 @@
+---
+name: 100-Lines-Of-Code
+desc: "This repository contains productive coded tools in less than #100LinesOfCode."
+site: https://github.com/josharsh/100LinesOfCode
+tags:
+- josharsh
```

Comment:
@josharsh Thats your username, why is it a Tag?

Reply:
corrected!